src/main/java/io/spring/initializr/generator/Dependency.java

```diff
@@ -21,7 +21,7 @@
  *
  * @author Andy Wilkinson
  */
-public class Dependency {
+public class Dependency implements Comparable<Dependency> {
```

Comment:
I think this makes sense. I was holding off doing it as I hadn't convinced myself that all users of `Dependency` would want the same ordering. We can always pull it back out again though.

Reply:
That makes sense. Hopefully we want dependencies to be ordered the exact same way in every build system.